annotated_types/__init__.py

```diff
@@ -129,8 +133,37 @@ class Le(BaseMetadata):
     le: SupportsLe
 
 
+class GroupedMetadata:
```

Comment:
Should this be

```suggestion
class GroupedMetadata(BaseMetadata):
```

?